app/templates/events/view/tickets/index.hbs

```diff
@@ -0,0 +1,107 @@
+<div class="ui three statistics">
+  <div class="statistic"><div class="value">4343</div><div class="label">{{t 'Orders'}}</div></div>
+  <div class="statistic"><div class="value">0.00</div><div class="label">{{t 'Sales(US $)'}}</div></div>
+  <div class="statistic"><div class="value">1</div><div class="label">{{t 'Tickets sold'}}</div></div>
+</div>
+<br>
+<div class="row">
+  <h4 class="ui header">Order Summary&nbsp;<span>By order status</span></h4>
+  <table class="ui celled table">
+    <thead>
+      <tr>
+        <th>{{t 'Order Status'}}</th>
+        <th class="right aligned">{{t 'Total Tickets'}}</th>
+        <th class="right aligned">{{t 'Total Orders'}}</th>
+        <th class="right aligned">{{t 'Total Sales'}} (US$)</th>
+        <th>{{t 'View Reports'}}</th>
+      </tr>
+      <tr>
+        <td class="ui grey inverted segment">
+          <span>Cancelled</span>
+        </td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td>View orders / View attendees</td>
+      </tr>
+      <tr>
+        <td class="ui yellow inverted segment">
+          <span>Pending</span>
+        </td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td>View orders / View attendees</td>
+      </tr>
+      <tr>
+        <td class="ui red inverted segment">
+          <span>Expired</span>
+        </td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td>View orders / View attendees</td>
+      </tr>
+      <tr>
+        <td class="ui blue inverted segment">
+          <span>Placed</span>
+        </td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td>View orders / View attendees</td>
+      </tr>
+      <tr>
+        <td class="ui green inverted segment"><span>Completed</span></td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td class="right aligned">0</td>
+        <td>View orders / View attendees</td>
+      </tr>
+    </thead>
+  </table>
+</div>
+<br>
+<div class="row">
+  <table class="ui celled table">
+    <thead>
+      <th scope="col">{{t 'Ticket Type'}}</th>
+      <th class="right aligned" scope="col">{{t 'Ticket Capacity Set'}}</th>
+      <th class="ui green inverted segment right aligned" scope="col" colspan="2">{{t 'Completed orders'}}</th>
+      <th class="ui blue inverted segment right aligned" scope="col" colspan="2">{{t 'Placed Orders'}}</th>
+      <th class="ui orange inverted segment right aligned" scope="col" colspan="2">{{t 'Pending Orders'}}</th>
+      <tr>
```

Comment:
@sumedh123 
- see how it's structured here https://github.com/fossasia/open-event-orga-server/blob/development/app/templates/gentelella/users/events/tickets/tickets.html#L174.  Make ticket type and ticket capacity set span two rows. 
- Then, center align the titles - 'Completed order', 'Placed orders', 'Pending orders'
- Then make `BY ORDER STATUS` => `By order status`